### PR TITLE
Added feature list

### PR DESCRIPTION
**Pull request by georgius1024**
## Added 
- 1
- 2
- 3

### CODE REVIEW

**Comment by Evgeni52** [comment]
test

File: README.md [hunk 1]
```diff
@@ -6,4 +6,9 @@
 
 ## Security regulations
 * R1
-* R2
\ No newline at end of file
+* R2
+
```
Comment (georgius1024) [inline README.md:10]: Why???

File: README.md [hunk 2]
```diff
@@ -6,4 +6,9 @@
...
+### Feature list
+* O1
```
Comment (georgius1024) [inline README.md:12]: Why???

File: README.md (hunk 1 above)
Comment (georgius1024) [reply README.md:10]: Because of

File: README.md (hunk 2 above)
Comment (georgius1024) [reply README.md:12]: OK

**Pull request merged**